src/org/labkey/targetedms/TargetedMSManager.java

```diff
@@ -2571,4 +2571,15 @@ else if (traceValue != null && values[i] >= traceValue)
                 .filter(qcMetricConfiguration -> qcMetricConfiguration.getTraceName() != null)
                 .collect(Collectors.toList());
     }
+
+    public static Map<String,Object> getQCFolderDateRange(Container container)
```

Comment:
What about having this return a Pair<Date, Date> instead? That would eliminate the need to cast and know the keys to the map.